frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -164,6 +170,76 @@ describe("scenarios > dashboard > dashboard drill", () => {
       .parent()
       .within(() => cy.findByText("foo"));
   });
+
+  it("should pass multiple filters for numeric column on drill-through (metabase#13062)", () => {
+    // go to admin > data model > sample dataset > reviews
+    cy.visit("/admin/datamodel/database/1/table/4");
+
+    // Set "Rating" Field type to: "Category" ("Score" is selected by default)
+    cy.findByText("Score").click();
+    // "Category" is not visible and any other method couldn't find it, including `Popover().contains("Category")`
+    cy.get(".ReactVirtualized__Grid")
+      .scrollTo("top")
+      .contains("Category")
+      .click();
+    // make sure the field updated before navigating away
+    cy.findByText("Category");
+
+    // go straight to simple question > reviews
+    cy.visit("/question/new?database=1&table=4");
+
+    // save the question
+    cy.findByText("Save").click();
+    cy.get(".Modal").within(() => {
+      cy.findByText("Save").click();
+    });
+    // and add it to a new dashboard
+    cy.findByText("Yes please!").click();
+    cy.findByText("Create a new dashboard").click();
+    cy.findByLabelText("Name")
+      .click()
+      .type("13062");
+    cy.findByText("Create").click();
+
+    // make sure we switched to the dashboard in edit mode
+    cy.findByText("You're editing this dashboard.");
+
+    // add filter
+    cy.get(".Icon-filter").click();
+    cy.findByText("Other Categories").click();
+    // and link it to the card
+    selectDashboardFilter(cy.get(".DashCard"), "Rating");
+
+    // save the dashboard and exit editing mode
+    cy.findByText("Save").click();
+    cy.findByText("You're editing this dashboard.").should("not.exist");
+
+    // add values to the filter
+    cy.findByText("Category").click();
+    popover().within(() => {
+      cy.findByText("5").click();
+      cy.findByText("4").click();
+    });
+    cy.findByText("Add filter").click();
+
+    // drill-through
+    cy.findByText("xavier").click();
+    cy.findByText("=").click();
+
+    cy.log("**Reported failing on Metabase 1.34.3 and 0.36.2**");
+    cy.findByText("Reviewer is xavier");
+    cy.findByText("Rating is equal to 2 selections");
+    // wait for data to finish loading
+    cy.get(".LoadingSpinner").should("not.exist");
+
+    cy.log("**Test the second case reported in this issue**");
+    // go back to the dashboard
+    cy.visit("/dashboard/2?category=5&category=4");
```

Comment:
The new dashboard you created on line 196 is ID=6
I don't know if you want to create a new dashboard or reuse an existing. If new, does it always get ID=6?
```suggestion
    cy.visit("/dashboard/6?category=5&category=4");
```

Reply:
If there's a `restore` before the test we should be able to count on newly created dashboard ids being the same for each run. However, that id will change when we update the default snapshot to add a database. It's probably best to save the ID earlier in the test or avoid using it with `cy.go('back')` or something.

Reply:
Good catch, @flamber 
It was dashboard 2 for me locally, and I didn't even caught that it's `6` in CI. It would be awesome if we could have the same **starting** dataset for both environments.
Btw, I first created a new dashboard and then tried to go back to it.

@paulrosenzweig - do we have plans to update the default "state" of the testing data set? Wouldn't that mess up with some other tests as well?

Reply:
> the same starting dataset for both environments

That should be the case already. The snapshots are regenerated on every run, but they should have the same number of dashboards created in the same order.

> do we have plans to update the default "state" of the testing data set?

I think we should expect changes. Inevitably, any tests using question 1 will fail if we totally change what that is in the snapshot, but I think we should write tests in a way that we can add more questions or dashboards to the snapshot data without breaking things.